### DISCUSSION TITLE
unable to connect to server

### DISCUSSION THREAD

**Discussion by WhirlyBird1912** [SpotX-Official/SpotX#235] [Errors]
hey! Just shifted from BlockTheSpot to SpotX and I installed the .bat file. But for some reason, when it opens command prompt it stays blank for 30 seconds or so and then displays this error message. Is there any fix?:
Invoke-WebRequest : Unable to connect to the remote server
At line:1 char:90
+ ... Tls12}; & {(Invoke-WebRequest -UseBasicParsing 'https://raw.githubuse ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand

**Comment by amd64fox** [comment]
change DNS from ISP default or use VPN
